src/components/Requirements.vue

```diff
@@ -248,10 +252,60 @@ export default Vue.extend({
     activateMinor(id: number) {
       this.displayedMinorIndex = id;
     },
+    createCourse(course: FirestoreSemesterCourse, isRequirementsCourse: boolean) {
+      this.$emit('createCourse', course, isRequirementsCourse);
+    },
     getRequirementsTooltipText() {
       return `<b>This is your Requirements Bar <img src="${clipboard}"class = "newSemester-emoji-text"></b><br>
           <div class = "introjs-bodytext">To ease your journey, we’ve collected a list of course
           requirements based on your college and major :)</div>`;
+    },
+    getCurrentType() {
+      let currentType;
+      const currentMonth = new Date().getMonth();
+      if (currentMonth === 0) {
+        currentType = 'WI';
+      } else if (currentMonth <= 4) {
+        currentType = 'SP';
+      } else if (currentMonth <= 7) {
+        currentType = 'SU';
+      } else {
+        currentType = 'FA';
+      }
+      return currentType;
+    },
```

Comment:
I remember seeing similar code elsewhere... Maybe a good time to move it into `utilities.ts`